app/connectors/itsastatus/ITSAStatusUpdateConnector.scala

```diff
@@ -78,11 +78,53 @@ class ITSAStatusUpdateConnector @Inject()(val httpClient: HttpClientV2, val appC
                     .map(failure => s"code: ${failure.code}, reason: ${failure.reason}")
                     .getOrElse("unknown reason")
 
-                log.error(s"response status: ${response.status}, message: $message")
+                logger.error(s"response status: ${response.status}, message: $message")
                 valid
               }
             )
         }
       }
   }
+
```

Comment:
If you wanted to, you can probably just use one function for both single and multi year and just put the single year ones in a list on its own. It'll save not having to have 2 functions doing pretty much the same thing